### PR TITLE
Feature/missing options

### PR DESCRIPTION
**Pull request by ankitrox**
### Customizer

1. Sharp corner option.
2. Remove story poster option.

### Widget

1. Add sharp corner option.
2. Archive Label option.

### Shortcode

1. Add sharp corner option.

### TinyMCE

1. Add archive label option.
2. Remove column number option from list and carousel.
3. Sharp corner option.

### CODE REVIEW

**Review by swissspidy — APPROVED** [review]
LGTM apart from the lint& unit test failures

**Review by spacedmonkey — APPROVED** [review]
👍

**Review by spacedmonkey — CHANGES_REQUESTED** [review]
Can you fix the test unit js and lint.

File: includes/Customizer.php [hunk 1]
```diff
@@ -320,67 +343,68 @@ public function register_customizer_settings( \WP_Customize_Manager $wp_customiz
...
 
 			$wp_customize->add_control(
-				self::STORY_OPTION . '[show_story_poster]',
+				self::STORY_OPTION . '[stories_archive_label]',
 				[
-					'type'            => 'checkbox',
+					'type'            => 'text',
 					'section'         => self::SECTION_SLUG,
-					'label'           => __( 'Show story poster', 'web-stories' ),
+					'label'           => __( 'Stories archive label', 'web-stories' ),
```
Comment (swissspidy) [inline includes/Customizer.php:402]:
```suggestion
					'label'           => __( 'Archive Link Label', 'web-stories' ),
```

That's what we use in the Customizer I think?

Let's make sure we use the same labels everywhere.

**Review by swissspidy — APPROVED** [review]

File: tests/phpunit/tests/Shortcode/Stories_Shortcode.php [hunk 2]
```diff
@@ -145,10 +146,11 @@ public function test_prepare_story_attrs() {
 			'date'                      => 'true',
 			'excerpt'                   => 'true',
 			'archive_link'              => 'randomtext',
-			'archive_label'             => 'true',
+			'archive_label'             => 'Stories',
```
Comment (spacedmonkey) [inline tests/phpunit/tests/Shortcode/Stories_Shortcode.php:149]:
```suggestion
			'archive_label'             => __( 'Stories', 'web-stories' ),
```

File: tests/phpunit/tests/Shortcode/Stories_Shortcode.php [hunk 3]
```diff
@@ -131,10 +131,11 @@ public function test_prepare_story_attrs() {
 			'show_date'                 => true,
 			'show_excerpt'              => true,
 			'show_story_archive_link'   => false,
-			'show_story_archive_label'  => true,
+			'stories_archive_label'     => 'Stories',
```
Comment (spacedmonkey) [inline tests/phpunit/tests/Shortcode/Stories_Shortcode.php:134]:
```suggestion
			'stories_archive_label'     => __( 'Stories', 'web-stories' ),
```

File: tests/phpunit/tests/Shortcode/Stories_Shortcode.php (hunk 2 above)
Comment (swissspidy) [reply tests/phpunit/tests/Shortcode/Stories_Shortcode.php:149]: This is a test file, no need for translation here.

File: tests/phpunit/tests/Shortcode/Stories_Shortcode.php (hunk 3 above)
Comment (swissspidy) [reply tests/phpunit/tests/Shortcode/Stories_Shortcode.php:134]: This is a test file, no need for translation here.

Comment (spacedmonkey) [reply tests/phpunit/tests/Shortcode/Stories_Shortcode.php:134]: 🤦

File: tests/phpunit/tests/Shortcode/Stories_Shortcode.php (hunk 2 above)
Comment (spacedmonkey) [reply tests/phpunit/tests/Shortcode/Stories_Shortcode.php:149]: 🤦

**Review by spacedmonkey — APPROVED** [review]
SGTM

**Pull request merged**